contrib/export/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/codec"
+	extypes "github.com/cosmos/cosmos-sdk/contrib/export/types"
+	"github.com/cosmos/cosmos-sdk/contrib/export/v036"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/types"
+	"log"
+)
+
+var (
+	migrationMap = extypes.MigrationMap{
+		"v0.36": v036.Migrate,
+	}
+	//source        string
+	target        string
+	importGenesis string
+)
+
+const (
+	// FlagSource will support multiple version upgrades
+	FlagSource = "source"
+)
+
+func migrateGenesisCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "genesis [v0.34] [~/my/genesis.json]",
+		Short: "Migrate genesis to a specified version",
+		Long:  `Migrate the source genesis into the target version and export it as standard output.`,
+		Args:  cobra.RangeArgs(2, 2),
+		RunE:  runMigrateCmd,
+	}
+
+	cmd.Flags().String(FlagSource, "", "The SDK version that exported the genesis")
+
+	return cmd
+}
+
+func runMigrateCmd(cmd *cobra.Command, args []string) (err error) {
+	if len(args) != 2 {
+		errMessage := "Wrong number of arguments provided, target version and genesis path are mandatory"
+		log.Println(errMessage)
+		return errors.New(errMessage)
+	}
+	// TODO: add support for multiple version upgrades by looping sequentially to migration functions
+	//_ := cmd.Flag(FlagSource)
```

Comment:
leftover comment ?

Reply:
the idea was to keep it as placeholder for the next version of this script, but we can drop it and re-include it next release

Reply:
I'll remove the commented code and move the TODO in the cmd definition